quodlibet/quodlibet/qltk/songsmenu.py

```diff
@@ -29,6 +29,17 @@
 from quodlibet.util.songwrapper import ListWrapper, check_wrapper_changed
 
 
+def confirm_song_removal_invoke(parent, songs):
+    songs = set(songs)
+    if len(songs) == 1:
+        title = _("Remove track: \"%s\" from library?"
+                  % next(iter(songs))['title'])
```

Comment:
Perhaps we should consider the possible-though-rare case where tracks don't have titles. We could default to the `~filename` in this case, perhaps, rather than display `""`